src/adapter/aws-lambda/handler.ts

```diff
@@ -162,11 +162,16 @@ export const streamHandle = <
   )
 }
 
+type HandleOptions = {
+  contentTypesAsBinary: string[]
```

Comment:
How about adding the following type or replacing `string[]`?

`(contentType: string) => boolean) | undefined`

This means the `HandleOptions` will be:

```ts
type HandleOptions = {
  contentTypesAsBinary: string[] | ((contentType: string) => boolean) | undefined
}
```

Or

```ts
type HandleOptions = {
  contentTypesAsBinary: ((contentType: string) => boolean) | undefined
}
```

You can use it like this:

```ts
const result = await processor.createResult(event, response, {
  contentTypesAsBinary: (contentType) => {
    return /application\/custom\-(a|b|c)/.test(contentType)
  },
})
```

This can handle the `contentType` more flexibly. I like passing a "callback" style for options 😃